src/Model/Subsite.php

```diff
@@ -263,7 +263,7 @@ public static function getSubsiteIDForDomain($host = null, $checkPermissions = t
 
             // ClassInfo::hasTable provides a cached table list
             if (!ClassInfo::hasTable($domainTableName)) {
-                // Table hasn't been created yet. Might be a dev/build, skip.
+                // Table hasn't been created yet. Might be bilding the db, skip.
```

Comment:
```suggestion
                // Table hasn't been created yet. Might be building the db, skip.
```

Reply:
I don't think it was worth holding back merging for a missing letter in a comment lol. But done.